wger/nutrition/forms.py

```diff
@@ -55,6 +55,8 @@ def __init__(self, *args, **kwargs):
 
 
 class BmiForm(forms.ModelForm):
+    height = forms.DecimalField(widget=Html5NumberInput(),
+                                max_value=999)
```

Comment:
Add here a ``label=_('Height (cm)')``, then the label is translated as well (that's the disadvantage of re-defining model fields, you have to duplicate a lot of things)